internal/orchestrator/bricks/testdata/my-app.source/python/main.py

```diff
@@ -0,0 +1,2 @@
+def main():
+    # empty file
```

Comment:
Is this a valid Python file? Shouldn't this be

```suggestion
    pass
```